fulp_modules/features/antagonists/bloodsuckers/code/bloodsucker/bloodsucker_conversion.dm

```diff
@@ -38,6 +38,10 @@
 	if(!master || (master == owner.current))
 		return TRUE
 	var/datum/antagonist/bloodsucker/bloodsuckerdatum = master.mind.has_antag_datum(/datum/antagonist/bloodsucker)
+	var/vassals_amount = bloodsuckerdatum.vassals.len
+	if(vassals_amount >= limit)
+		to_chat(owner.current, span_danger("Your are unable to bring more than 3 vassals under your control!"))
```

Comment:
```suggestion
		to_chat(owner.current, span_danger("Your are unable to bring more than [limit] vassals under your control!"))
```

Reply:
in this case if you take my suggestion you'll have to chance it to vassal_limit

Reply:
Thanks